ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4733,3 +4733,12 @@ HWRWISOVPM:
   - heating_return_water_isolation_valve_percentage_sensor
   implements:
   - MONITORING
+
+DDPM:
```

Comment:
DDPC. please change. This is discharge air damper control, not simply monitoring.